pkg/services/gotify/gotify_test.go

```diff
@@ -0,0 +1,26 @@
+package gotify
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestGotify(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Shoutrrr Gotify Suite")
+}
+
+var _ = Describe("the Gotify plugin", func() {
+	It("should work", func() {
```

Comment:
Maybe this could be a little more descriptive? What we're actually testing is that the url builder works.

Reply:
Completely agree. This was directly "stolen" from the teams part. I also added a few more tests.

Reply:
Awesome! Thanks!